### PR TITLE
I fine paragraph

### PR DESCRIPTION
**Pull request by lucas-tester**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 <body>
     <h1>Hi there, mister!</h1>
     <p class="text">How are you? Are you having a good time?</p>
+    <p>I fine</p>
```
Comment (lucas-segundo) [inline index.html:13]: Something wrong here! @lucas-tester

**Review by lucas-segundo — CHANGES_REQUESTED** [review]
@lucas-tester you must fix the <p>

File: index.html (hunk 1 above)
Comment (lucas-tester) [reply index.html:13]: ok, I will fix it.

Comment (lucas-tester) [reply index.html]: Fixed with commit 02b3f38

**Review by lucas-segundo — APPROVED** [review]
ok

**Pull request merged**